File: CPP/Easy/Remove-Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
       int previous(0);
       while (previous<nums.size()){
           if (nums[previous]==val){
               nums.erase( nums.begin( )+previous);
           }
           else previous+=1;
           
       }
       return nums.size();
    }
};

File: Python/Easy/Remove-Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        index = []
        for i in range(0,len(nums)):
            if nums[i] == val:
                index.append(i)
        index = index[::-1]
        for i in index:
            del nums[i]   
        return len(nums)
